### DISCUSSION TITLE
Appreciation for the logseq-bidi extension

### DISCUSSION THREAD

**Discussion by mrostd** [dobidi/logseq-bidi#17] [General]
I am writing to express my sincere gratitude for your work on the logseq-bidi extension. As a user who frequently writes in right-to-left languages, this extension has significantly improved my experience with Logseq.
Before discovering your extension, I was struggling with the lack of native right-to-left support in Logseq. Your extension has effectively addressed this issue, making it much easier for me to take notes in my native language.
Your work is greatly appreciated. Please keep up the excellent work!

**Comment by ahangarha** [comment]
Thank you for sharing your kind words. 🌺 

I am happy that this plugin makes your knowledge management journey easier. That is the best reward one may receive.

I look forward to your feedback and possibly contribution.

Bests

**Comment by azizabogoda** [comment]
Great job on the plugin!
